micrometer-core/build.gradle

```diff
@@ -19,6 +19,7 @@ dependencies {
     compileOnly libs.hdr.histogram
 
     implementation mn.reactor
+    implementation mn.micronaut.retry
```

Comment:
this should be compileOnly

Reply:
resolved via https://github.com/micronaut-projects/micronaut-micrometer/pull/547/commits/572fbc384d42e39a341cc85b04d144bfdb0176a5